tests/system_tests/test_blueapi_system.py

```diff
@@ -29,15 +32,66 @@
 
 _DATA_PATH = Path(__file__).parent
 
+# Instructions to run the system tests:
+# Step 1: Ensure ActiveMQ is running:
+#   podman run -it --rm --net host rmohr/activemq:5.15.9-alpine
+#
+# Step 2: Set the required environment variables:
+#   export TOKEN_URL="https://example.com/token"
+#   export PKCE_AUTHENTICATION_URL="https://example.com/auth"
+#
+# Step 3: Start the BlueAPI server with valid configuration:
+#   blueapi -c tests/unit_tests/example_yaml/valid_stomp_config.yaml serve
+#
+# Step 4: Run the system tests using tox:
+#   tox -e system-test
+#
+# Note: The system tests will be executed in the CI pipeline after resolving:
+#   https://github.com/DiamondLightSource/blueapi/issues/630
+
+
+@pytest.fixture
+def oauth_server() -> OAuthServerConfig:
+    return OAuthServerConfig(oidc_config_url="https://example.com")
+
+
+@pytest.fixture
+def oauth_client() -> CLIClientConfig:
+    return CLIClientConfig(
+        client_id="example-client",
+        client_audience="example",
+        token_file_path=Path("example-token-file"),
+    )
+
 
 @pytest.fixture
-def client() -> BlueapiClient:
+def client_without_auth() -> BlueapiClient:
     return BlueapiClient.from_config(config=ApplicationConfig())
 
 
 @pytest.fixture
-def client_with_stomp() -> BlueapiClient:
-    return BlueapiClient.from_config(config=ApplicationConfig(stomp=StompConfig()))
+def client_with_stomp(
+    oauth_server: OAuthServerConfig, oauth_client: CLIClientConfig
+) -> BlueapiClient:
+    return BlueapiClient.from_config(
+        config=ApplicationConfig(
+            stomp=StompConfig(),
```

Comment:
If we use the rmq image above, 
```suggestion
            stomp=StompConfig(user="guest", password="guest"),
```